src/main/java/org/cascadebot/cascadebot/utils/buttons/ButtonGroup.java

```diff
@@ -90,7 +95,7 @@ public void setMessage(long id) {
     }
 
     public void handleButton(Member clicker, TextChannel channel, Message buttonMessage, MessageReaction.ReactionEmote emote) {
-        for (Button button : buttons) {
+        for (Button button : this.getButtons()) {
```

Comment:
Why the change? Doesn't make a difference just wondering why

Reply:
It actually does make a difference. This allows it to get the correct list of buttons for say persistent buttons. Persistent buttons actually doesn't use the standard button list at all.

Reply:
Oh I see, this is used so it gets the button list from the overrided method instead. Got it 👌